src/transaction/lock_table.c

```diff
@@ -117,75 +123,81 @@ LOCK_COMPATIBILITY lock_Comp[11][11] = {
  * column : current lock mode (granted lock mode)
  * row    : request lock mode
  * -----------------------------------------------------------------------------------------------
- *         | N/A  NON2PL   NULL  SCH-S     IS      S     IX    SIX      U      X  SCH-M
+ *         | N/A  NON2PL   NULL  SCH-S     IS      S     IX    SIX      U      X  SCH-M   BU_LOCK
```

Comment:
please also change `BU_LOCK` with `BU` and relocate this between `IX` and `SIX`.